tutorials/dataframe/df106_HiggsToFourLeptons.C

```diff
@@ -0,0 +1,281 @@
+/// \file
+/// \ingroup tutorial_dataframe
+/// \notebook -draw
+/// The Higgs to four lepton analysis from the ATLAS Open Data release of 2020, with RDataFrame.
+///
+/// This tutorial is the Higgs to four lepton analysis from the ATLAS Open Data release in 2020
+/// (http://opendata.atlas.cern/release/2020/documentation/). The data was taken with the ATLAS detector
+/// during 2016 at a center-of-mass energy of 13 TeV. The decay of the Standard Model Higgs boson
+/// to two Z bosons and subsequently to four leptons is called the "golden channel". The selection leads
+/// to a narrow invariant mass peak on top a relatively smooth and small background, revealing
+/// the Higgs at 125 GeV.
+/// The analysis is translated to an RDataFrame workflow processing about 300 MB of simulated events and data.
+///
+/// Lepton scale factors are applied to leptons in MC simulations to correct for the differences in the
+/// reconstruction, identification and trigger mechanisms used for MC samples and real data.
+/// Systematic uncertainties for those MC scale factors are evaluated and the Vary function of RDataFrame is used to
+/// scale the variations to the final four leptons mass distribution.
```

Comment:
```suggestion
/// Lepton selection efficiency corrections ("scale factors") are applied to simulated samples to correct for the
/// differences in the trigger, reconstruction, and identification efficiencies in simulation compared to real data.
/// Systematic uncertainties for those scale factors are evaluated and the Vary function of RDataFrame is used to
/// propagate the variations to the final four leptons mass distribution.
```

I would avoid using jargon like "lepton scale factors" or "MC". I guess for the "scale factors" it's unavoidable, but it would be good to first explain this in other words.

Also "correct for the differences in the reconstruction, identification and trigger mechanisms" is not completely correct. The idea is that we are using the same "mechanisms" in simulation in data, but then these have different *efficiencies* because the simulation is not 100 % accurate. And maybe list as "trigger, reconstruction, and identification" to be in "chronological" order.